nexus/utils/index.ts

```diff
@@ -212,32 +223,39 @@ export const updatedThreadPositions = (
       breakPoints.add(idx)
     }
 
-    (tree.children || []).map(recur)
+    ;(tree.children || []).map(recur)
   }
 
   newDoc.map(recur)
 
-  for (let breakPoint of breakPoints) {
-    for (let ti = 0; ti<threadsRepr.length; ti++) {
+  for (const breakPoint of breakPoints) {
+    for (let ti = 0; ti < threadsRepr.length; ti++) {
       const t = threadsRepr[ti]
       if (t[0] < breakPoint && t[1] > breakPoint) {
-          t[0] = -1
-          t[1] = -1
+        t[0] = -1
+        t[1] = -1
       }
     }
   }
 
-  return threads.map(thread => {
+  return threads.map((thread) => {
     const [startIndex, endIndex] = threadsRepr.find(([_, __, id]) => id === thread.id) || [0, 0, 0]
     return {
       ...thread,
       startIndex,
-      endIndex
+      endIndex,
     }
   })
 }
 
-export const processEditorDocument = (document: NodeType[]) => {
+export const processEditorDocument = (
+  document: NodeType[],
+): {
+  body: string
+  bodySrc: string
+  excerpt: string
+  readTime: number
+} => {
```

Comment:
Same question here with regards to not using our types if we can't rely on the TS compiler